controllers/disruption_controller.go

```diff
@@ -1046,10 +1050,34 @@ func (r *DisruptionReconciler) recordEventOnTarget(instance *chaosv1beta1.Disrup
 }
 
 // SetupWithManager setups the current reconciler with the given manager
-func (r *DisruptionReconciler) SetupWithManager(mgr ctrl.Manager) error {
+func (r *DisruptionReconciler) SetupWithManager(mgr ctrl.Manager, kubeInformerFactory kubeinformers.SharedInformerFactory) error {
+	podToDisruption := func(c client.Object) []reconcile.Request {
+		// podtoDisruption is a function that maps pods to disruptions. it is meant to be used as an event handler on a pod informer
```

Comment:
Should we track the number of calls to the informer function and send it as a metric?

Reply:
Are you okay with the new metric and tags?